packages/rest/lib/resource.cadl

```diff
@@ -78,8 +78,8 @@ interface ResourceUpdate<TResource, TError> {
 @doc("Resource deleted successfully.")
 model ResourceDeletedResponse {
   @doc("The status code.")
-  @header
-  statusCode: 200;
+  @statusCode
+  _: 200;
```

Comment:
This feels a bit clunky, but I see the advantages of having a property versus a pure decorator, esepcially when considering versioning an API.

I wonder if we shouldn't add new cadl type / alias for status codes that we would expect spec writers to use for the property type

Reply:
I agree that this is clunky.  Perhaps we put this on the agenda for a future design session.